client/spec/pom.xml

```diff
@@ -14,29 +14,59 @@
   ~ See the License for the specific language governing permissions and
   ~ limitations under the License.
   -->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <parent>
         <groupId>org.eclipse.microprofile.graphql</groupId>
         <artifactId>microprofile-graphql-client</artifactId>
         <version>1.1-SNAPSHOT</version>
         <relativePath>../</relativePath>
     </parent>
-    
+
     <artifactId>microprofile-graphql-client-spec</artifactId>
     <packaging>pom</packaging>
-    
+
     <name>MicroProfile GraphQL Client :: Specification</name>
     <description>Client-side code-first GraphQL APIs for MicroProfile :: Specification</description>
-    
+
+    <dependencies>
+        <!-- Dynamic Client API -->
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>microprofile-graphql-client-api</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <!-- Needed to generate the class diagram for the client API -->
+        <dependency>
+            <groupId>jakarta.json</groupId>
+            <artifactId>jakarta.json-api</artifactId>
+            <version>1.1.6</version>
```

Comment:
Can this be scope provided ?

Reply:
Tested it with "provided" and it's OK. I will update.